stellar-contract-env-common/src/tuple.rs

```diff
@@ -5,29 +5,31 @@ use crate::{
     TryFromVal,
 };
 
+use core::marker::PhantomData;
+
 macro_rules! impl_for_tuple {
     ( $count:literal $($typ:ident $idx:tt)+ ) => {
         impl<E: Env, $($typ),*> TryFrom<EnvVal<E, RawVal>> for ($($typ),*)
         where
             $($typ: TryFrom<EnvVal<E, RawVal>>),*
         {
-            type Error = ConversionError;
+            type Error = ConversionError<($($typ),*)>;
```

Comment:
If you merge main into your branch I think you'll need to change this to:
```suggestion
            type Error = ConversionError<($($typ,)*)>;
```
A fix for tuples was merged so that 1-ary tuples are supported, which requires the trailing comma to distinguish a value in parenthesis from a tuple that uses parenthesis.

Reply:
Fixed.